ironfish/src/network/messages/pooledTransactionsRequest.ts

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import { SerializedTransaction, TransactionHash } from '../../primitives/transaction'
+import { NetworkMessageType } from '../types'
+import { Direction, RpcNetworkMessage } from './rpcNetworkMessage'
+
+export class PooledTransactionsRequest extends RpcNetworkMessage {
```

Comment:
I think our other RPC message files are just called e.g. `getBlocks` and `getBlockHashes` -- maybe this could be `pooledTransactions` or `getPooledTransactions` to match the naming format?